sessionctx/variable/sysvar.go

```diff
@@ -136,7 +136,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeGlobal | ScopeSession, "ndb_index_stat_option", ""},
 	{ScopeGlobal | ScopeSession, OldPasswords, "0"},
 	{ScopeNone, "innodb_version", "5.6.25"},
-	{ScopeGlobal, MaxConnections, "151"},
+	{ScopeGlobal, MaxConnections, strconv.FormatInt(DefMaxConnections, 10)},
```

Comment:
Maybe we should add a upgradeVer function to upgrade it to a large value, e.g. 1000.

Reply:
PTAL @coocood